Mathlib/RingTheory/Extension/Presentation/Core.lean

```diff
@@ -179,10 +179,10 @@ private lemma hom_comp_inv :
   ext x
   simp
 
+attribute [-ext] MvPolynomial.algHom_ext MvPolynomial.algHom_ext' in
 private lemma inv_comp_hom :
     (P.tensorModelOfHasCoeffsInv R₀).comp (P.tensorModelOfHasCoeffsHom R₀) = AlgHom.id R _ := by
-  ext x
-  obtain ⟨x, rfl⟩ := Ideal.Quotient.mk_surjective x
```

Comment:
Which `ext` lemma is being applied on master? I'm asking to understand why we need to remove the attribute from the scope in this declaration now.